src/components/snapshot/overview/SnapshotWorkspaceAccordionView.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { TdrState } from 'reducers';
+import { SnapshotWorkspaceEntry } from 'models/workspaceentry';
+import { CustomTheme, List, ListSubheader, Typography } from '@mui/material';
+import { createStyles, withStyles, WithStyles } from '@mui/styles';
+import { InaccessibleWorkspacePolicyModel, WorkspacePolicyModel } from 'generated/tdr';
+import ManageWorkspacesModal from './ManageWorkspacesModal';
+import SnapshotWorkspaceEntriesList from './SnapshotWorkspaceAccordionListEntries';
+
+const styles = (theme: CustomTheme) =>
+  createStyles({
+    snapshotWorkspaceList: {
+      width: '100%',
+    },
+    nullValue: {
+      fontStyle: 'italic',
+      textColor: theme.palette.primary.dark,
+      color: theme.palette.primary.dark,
+    },
+    buttonDescription: {
+      marginLeft: '0.1em',
+      display: 'inline-flex',
+    },
+  });
+
+type SnapshotWorpsaceFunctionProps = {
+  removeWorkspace: any;
+};
+type SnapshotWorkspaceViewProps = SnapshotWorpsaceFunctionProps &
+  StateProps &
+  WithStyles<typeof styles>;
+
+function getDefaultTitle(id: string | undefined): string {
+  if (id) {
+    return `Workspace ID ${id}`;
+  }
+  return '<UNKNOWN WORKSPACE>';
```

Comment:
A little extra insurance, looks like?

Reply:
Yes, leftover from the days of trusting the generated model.  ;-)